whatwg-streams/index.d.ts

```diff
@@ -31,7 +31,7 @@ declare class ReadableStream {
     cancel(reason: string): Promise<void>;
     getReader(): ReadableStreamDefaultReader;
     getReader({ mode }: { mode: "byob" }): ReadableStreamBYOBReader;
-    pipeThrough<T extends ReadableStream>({ writable, readable }: { writable: WritableStream, readable: T }): T;
+    pipeThrough<T extends ReadableStream>({ writable, readable }: { writable: WritableStream, readable: T }, options?: any): T;
```

Comment:
`pipeThrough` internally calls `pipeTo(writable, options)` so the `options` parameter should have the same interface of the second argument of `pipeThrough`.

```ts
{ preventClose?: boolean, preventAbort?: boolean, preventCancel?: boolean }
```

We probably should make a named interface for this, and the name may be `PipeOptions` or something.